src/incubator/expandableOverlay/index.tsx

```diff
@@ -68,22 +70,34 @@ const ExpandableOverlay = (props: ExpandableOverlayProps, ref: any) => {
     ...others
   } = props;
   const [visible, setExpandableVisible] = useState(false);
+  const containerRef = useRef(null);
+
+  const focusAccessibility = useCallback(() => {
+    const reactTag = findNodeHandle(containerRef.current);
+    if (reactTag) {
+      AccessibilityInfo.setAccessibilityFocus(reactTag);
+    }
+  }, [containerRef]);
```

Comment:
AFAIK you should not put a ref as a dependency, it won't trigger a change
```suggestion
  }, []);
```

Reply:
Mistake. Removed 👌